src/components/pages/Users/EditUserPanel/EditUserPanel.test.tsx

```diff
@@ -544,3 +554,98 @@ test("should handle errors when updating entitlements", async () => {
     }),
   ).toBeInTheDocument();
 });
+
+test("should change user details", async () => {
+  const invalidateQueries = vi.fn();
+  vi.spyOn(reactQuery, "useQueryClient").mockReturnValue({
+    invalidateQueries,
+  } as unknown as reactQuery.QueryClient);
+  let patchResponseBody: string | null = null;
+  let patchDone = false;
+  // eslint-disable-next-line @typescript-eslint/no-misused-promises
+  mockApiServer.events.on("request:start", async ({ request }) => {
+    const requestClone = request.clone();
+    if (
+      requestClone.method === "PUT" &&
+      requestClone.url.endsWith("/identities/user1")
+    ) {
+      patchResponseBody = await requestClone.text();
+      patchDone = true;
+    }
+  });
+  const {
+    result: { findNotificationByText },
+  } = renderComponent(
+    <EditUserPanel
+      user={mockUser}
+      onUserUpdate={() => void invalidateQueries({ queryKey: ["/identities"] })}
```

Comment:
There's no need to call `invalidateQueries` here (and could result in false positives). Instead this should be a vi.fn() and then we can check if that function was called.